test/byContract/IexecPoco/06_claim.test.ts

```diff
@@ -144,12 +144,8 @@ describe('IexecPoco2#claim', async () => {
             .withArgs(sponsor.address, taskPrice)
             .to.emit(iexecPoco, 'Seize')
             .withArgs(scheduler.address, schedulerTaskStake, taskId)
-            .to.emit(iexecPoco, 'Transfer')
-            .withArgs(iexecPoco.address, kittyAddress, schedulerTaskStake)
```

Comment:
Do we have the same here ? https://github.com/iExecBlockchainComputing/PoCo/blob/d443cc467b8603eee00aa02a62fcd958da547bbe/test/byContract/IexecPoco/04_finalize.test.ts#L519

Reply:
Thank you very much, I screened it first then moved on since test were OK.
I found a framework issue which forces us to [check token balance changes before checking events](https://github.com/iExecBlockchainComputing/PoCo/pull/167/commits/0ac30f0d9a6fc0ee71d0fd61bfa9b6a3fed2228a), otherwise event checking never works ..!
We need to update that in both projects in next PRs (FYI @iExecBlockchainComputing/team-blockchain)

Reply:
Thanks for the point, we should keep in mind this !